src/utils/sort.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright © 2020 The PES Open Source Team pesos@pes.edu
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"sort"
+	"strconv"
+)
+
+// SortData helps sort table rows. It sorts the table based on values given in the sortIdx column and sorts ascending if sortAsc is true. sortCase is set to identify the set of 'less' functions to use to sort the selected column by.
+func SortData(data [][]string, sortIdx int, sortAsc bool, sortCase string) {
+
+	// Define less functions
+	intSort := func(i, j int) bool {
+		x, _ := strconv.Atoi(data[i][sortIdx])
```

Comment:
Would ignoring the error be a good idea? Not sure of a case when such an error would occur, but if it does, ignoring it might not be the best thing. 

Since such a function assumed that its a valid `int` or a valid `float`, it might help to enforce this assumption by passing converted data to the sort funtion, or a wrapper that would potentially help handling ignored errors. WDYT?

Reply:
You can't really give converted data to the `less` functions, you can only ensure data coming into the `sortData()` is clean. But even in the case that conversion fails, zero values are returned and the data will stay the same post sort. So I feel it's alright to ignore the error here. Handling it would cause unnecessary complexity to a simple function I feel.

Reply:
Fair enough 👍